config.yaml

```
@@ -52,6 +52,7 @@ default_artifacts:
     - metal4k
     - live
     - openstack
+    - hashlist-exp
```

```suggestion
    - experimental-hashlist
```

If we end up doing https://github.com/coreos/coreos-assembler/pull/3248#issuecomment-1332758075